tests/clvm/coin_store.py

```diff
@@ -69,12 +70,15 @@ def validate_spend_bundle(
         if result.error is not None:
             raise BadSpendBundleError(f"condition validation failure {Err(result.error)}")
 
-        ephemeral_db = dict(self._db)
+        ephemeral_db = {}
+        for k, v in self._db.items():
+            ephemeral_db[bytes_to_CoinID(k)] = v
```

Comment:
```suggestion
        ephemeral_db = {bytes_to_CoinID(k): v for k, v in self._db.items()}
```

Reply:
would that work?

Reply:
did you explore changing the type of `self._db` instead?

Reply:
>    ephemeral_db = {bytes_to_CoinID(k): v for k, v in self._db.items()}

Hey nice, that works just like a list comprehension! Sweet.

Reply:
> did you explore changing the type of `self._db` instead?

Since this is `tests.clvm.coin_store` and not `chia.full_node.coin_store`, I will definitely take this suggestion, thanks.